modules/nixos/sddm.nix

```diff
@@ -13,12 +13,16 @@ let
     versionAtLeast
     ;
   cfg = config.services.displayManager.sddm.catppuccin;
-  enable = cfg.enable && config.services.displayManager.sddm.enable;
+  enable =
+    versionAtLeast ctp.getModuleRelease minVersion
```

Comment:
Why is this extra version check needed? You already can't enable `cfg` on versions under `minVersion`.

Maybe it's better to place the `mkVersionedOpts` on `options.services.displayManager.sddm.catppuccin` so you get this error 
![image](https://github.com/catppuccin/nix/assets/44153531/2ae8fa95-43cf-467e-a90a-7c6ca7e39b40)
instead of this 
![image](https://github.com/catppuccin/nix/assets/44153531/b97934da-83fd-4e8f-807e-fb14f420a9b5)
when trying to enable `services.displayManager.sddm` with a version under `minVersion`

Reply:
I didn't test it, but theoretically, without the version check, `enable` depends on `config.services.displayManager.sddm.catppuccin` even when it's not defined and not enabled